src/PolyhedralGeometry/PolyhedralComplex/properties.jl

```diff
@@ -0,0 +1,410 @@
+@doc Markdown.doc"""
+    ambient_dim(PC::PolyhedralComplex)
+
+Return the ambient dimension of `PC`.
+
+# Examples
+```jldoctest
+julia> IM = IncidenceMatrix([[1,2,3],[1,3,4]])
+2×4 IncidenceMatrix
+[1, 2, 3]
+[1, 3, 4]
+
+
+julia> V = [0 0; 1 0; 1 1; 0 1]
+4×2 Matrix{Int64}:
+ 0  0
+ 1  0
+ 1  1
+ 0  1
+
+julia> PC = PolyhedralComplex(IM, V)
+A polyhedral complex in ambient dimension 2
+
+julia> ambient_dim(PC)
+2
+```
+"""
+ambient_dim(PC::PolyhedralComplex) = Polymake.fan.ambient_dim(pm_object(PC))
```

Comment:
I would recommend to add type assertions to these methods to improve type stability (see  also PR #927). E.g.:
```suggestion
ambient_dim(PC::PolyhedralComplex) = Polymake.fan.ambient_dim(pm_object(PC))::Int
```